activestorage/test/analyzer/image_analyzer_test.rb

```diff
@@ -29,4 +29,13 @@ class ActiveStorage::Analyzer::ImageAnalyzerTest < ActiveSupport::TestCase
     assert_equal 792, metadata[:width]
     assert_equal 584, metadata[:height]
   end
+
+  test "analyzing an unsupported image type returns nil width and height metadata" do
+    blob = create_file_blob(filename: "bad_file.bad", content_type: "image/bad_type")
```

Comment:
Create a new fixture here, but we could also mock and stub the assertions.

Reply:
Use `create_blob(data: "", ...)` to avoid the empty fixture.